### ISSUE TITLE
Unreliable durability system

### ISSUE THREAD

**Issue by Winqov** [DorkixAzIgazi/armored-elytra#6]
**Describe the bug**
There's something wrong with the durability coding. Apart from not getting durability damage from hostile sources (see https://github.com/DorkixAzIgazi/armored-elytra/issues/5), there are other issues.

When you split an armored elytra, the items durability will return to their state when crafted (see screenshots).

**To Reproduce**
Craft armored elytra, preferably from damaged armor and elytra (so it's easier to spot the issue)
Damage the armored elytra (currently only possible when flying)
Split the armored elytra, the result items have the same durability from the start
This way, you can "repair" them just by splitting and crafting again

**Expected behavior**
Durability works.

**Screenshots**
![2024-10-19_12 34 32](https://github.com/user-attachments/assets/e588415e-f505-4cdd-9d4e-70269fa90771)
![2024-10-19_12 34 34](https://github.com/user-attachments/assets/7dd5b938-f8a0-41c4-8f78-435d3bd09365)
![2024-10-19_12 34 47](https://github.com/user-attachments/assets/7fce5cb8-bf5f-447a-8bef-10790dbeac65)
![2024-10-19_12 35 33](https://github.com/user-attachments/assets/eb430261-085b-4b90-b252-6e6e9e61d63e)
![2024-10-19_12 35 51](https://github.com/user-attachments/assets/eb753738-e634-412e-a588-94740e779a3f)
![2024-10-19_12 35 55](https://github.com/user-attachments/assets/4aeddac1-9349-439a-8576-c2c865247435)

**Desktop (please complete the following information):**
 - Minecraft 1.21.1
 - armored-elytra-1.1.0.jar
 - fabric-api-0.106.0+1.21.1.jar

**Additional context**
Sorry, I didn't know how to title this issue. Feel free to change it to something more descriptive for you. I really like this mod, but in this state, it is very unusable in a serious playthrough.

**Comment by DorkixAzIgazi** [comment]
Good point both #5 and #6, I might be able to fix these in the 1.21.4 version. The difficulty with this one is that, when crafting the item back into two parts how do you scale the taken durability? The elytra and the armor has different max durabilty values. I could do it by percent. What do you think of that? @Winqov
